gh-pages/content/user-guides/lib-author/configuration/targets/go.md

```diff
@@ -3,15 +3,31 @@
 !!! danger
     The **go** target is currently unstable and not suitable for production use.
 
-To enable go package generation, add the **go** key with an empty object to the jsii targets configuration.
+To enable go package generation, add the `go` key to the jsii targets configuration:
 
-This will add generated go package code to your specified `outDir` for testing and experimentation.
+- `packageName` (optional) - The name of the Go package name. If this is not
```

Comment:
This is confusing to me as go modules can have multiple packages. I would rename to `rootPackageName` and explain what exactly it contains (i.e types that are directly exported from the entry point?)

Reply:
Are you referring to submodules? Because I believe it should be possible to specify the package name for each submodule by placing a jsii config file near the module root. @RomainMuller will just JUST WORK for Go or do we need extra support for this?

Reply:
I actually blanked out and thought every subdirectory directory will translate to its own go package, but of course it needs to be explicitly exported from `index.ts`. 

 > Added a test which verifies that this works for submodules, in which case, packageName is a better name because it can be used recursively.

Agree. I didn't know we can specify a nested jsii config, TIL 👍